dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -59,6 +59,16 @@
   @eyes.check_window(identifier, MATCH_TIMEOUT)
 end
 
+# Temporary step used by tests that have diffs between when they execute in the test environment vs a CircleCI
+# build, so that we can enable Applitools / Github integration.
+# TODO: (suresh) Remove this step and revert tests to call "I see now difference for" when the test has been modified
+# to work identically in CircleCI environments.
+And(/^I see no difference in test environment for "([^"]*)"$/) do |identifier|
+  next if CDO.disable_all_eyes_running || ENV[IS_CIRCLE]
+
+  @eyes.check_window(identifier, MATCH_TIMEOUT)
+end
```

Comment:
+1 to Andrew's comment, please combine this with the previous step using the `(...)?` pattern.